components/App.js

```diff
@@ -42,6 +53,98 @@ export const onclickBindings = {
       menu,
     });
   },
+  'threeD-social-tradeBtn': (e) => {
+    const { menu } = getState();
```

Comment:
These braces do not follow lint rules.

Reply:
if your linter is different then normal JS rules then need to lint with script